### PR TITLE
update c7n_org to exit(1) on failure to provision policy

### PR DESCRIPTION
**Pull request by aardvarq**
addresses https://github.com/capitalone/cloud-custodian/issues/2656

### CODE REVIEW

**Comment by kapilt** [comment]
related #2659 .. the primarily delta between the two prs, is this one will raise/exit eagerly from a process while the other one will wait till all accounts are processed and exit 1 if an error is detected.

File: tools/c7n_org/c7n_org/cli.py [hunk 1]
```diff
@@ -486,7 +486,7 @@ def run_account(account, region, policies_config, output_path,
                 log.error(
                     "Exception running policy:%s account:%s region:%s error:%s",
                     p.name, account['name'], region, e)
-                continue
+                raise SystemExit(1)
```
Comment (kapilt) [inline tools/c7n_org/c7n_org/cli.py]: not clear this is actually going to do the intended effect, c7n-org runs multi-process, this just kills the child, not the parent afaics.

**Comment by aardvarq** [comment]
So we want the exit happening further up the stack.... To your point, it is successfully terminating my CI job due to exit status but scheduled processes continue to spawn when we'd prefer that they do not.

**Comment by kapilt** [comment]
merged in #2659 which addresses this same issue of exiting 1 on any policy error (but not stopping execution of policies/accounts, till all accounts are attempted).

**Pull request closed without merging**